site/source/docs/api_reference/html5.h.rst

```diff
@@ -419,13 +419,13 @@ Struct
   .. c:member:: long targetX
      long targetY
 
-    These fields give the mouse coordinates mapped relative to the coordinate space of the target DOM element receiving the input events (Emscripten-specific extension).
+    These fields give the mouse coordinates mapped relative to the coordinate space of the target DOM element receiving the input events (Emscripten-specific extension; coordinates are rounded down to whole units).
```

Comment:
How about `down to the nearest integer`?

Reply:
I wasn't sure of the best wording. It's CSS units without the screen scaling, and I was trying to be concise. I'll make a quick edit.